api/CustomElementRegistry.json

```diff
@@ -0,0 +1,65 @@
+{
+  "api": {
+    "CustomElementRegistry": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CustomElementRegistry",
+        "description": "The CustomElementRegistry interface provides methods for registering custom elements and querying registered elements.",
+        "support": {
+          "chrome": {
+            "version_added": "33",
+            "notes": "Chrome 36+/Opera 20+ implemented a previous version of Custom Elements (v0) that used `.registerElement()`"
+          },
+          "firefox": {
+            "version_added": true,
+            "notes": "Enabled through the `dom.webcomponents.enabled` preference in `about:config`"
+          },
+          "safari": {
+            "version_added": "10.1",
+            "notes": "Supports 'Autonomous custom elements' but not 'Customized built-in elements'"
+          },
+          "edge": {
+            "version_added": false,
+            "notes": "Under consideration"
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": "41",
+            "notes": "Supports 'Autonomous custom elements' but not 'Customized built-in elements'"
+          },
+
+
+          "chrome_android": {
+            "version_added": "61",
+            "notes": "Supports 'Autonomous custom elements' but not 'Customized built-in elements'"
```

Comment:
At first look, I think customized built-in elements should be a subfeature here.

Reply:
@teoli2003 not sure I follow. I merely used `GainNode` as a reference point so not sure I know how to do this as this was not in there. Any examples?